edk2toolext/environment/version_aggregator.py

```diff
@@ -52,6 +52,9 @@ def GetAggregatedVersionInformation(self):
         """
         return copy.deepcopy(self.Versions)
 
+    def Reset(self):
```

Comment:
would love to see unit test code for this now that you touched it.  :)

Reply:
Haha- that's fair enough

Reply:
First crack is done